app/assets/javascripts/discourse/app/controllers/full-page-search.js

```diff
@@ -308,6 +310,18 @@ export default Controller.extend({
       : "search-info";
   },
 
+  @discourseComputed("model.posts", "additionalSearchResults")
+  searchResultPosts(posts, additionalSearchResults) {
+    if (additionalSearchResults && additionalSearchResults?.list?.length > 0) {
```

Comment:
```suggestion
  @discourseComputed("model.posts", "additionalSearchResults.list")
  searchResultPosts(posts, additionalResultsList) {
    if (additionalResultsList?.length > 0) {
```

Very minor, but I don't think we need both `additionalSearchResults &&` and `additionalSearchResults?` in this clause. In fact, maybe we can take this a step further as per above (the reference to `additionalSearchResults.list` in line 317 below would need to be updated as well).